xonsh/platform.py

```diff
@@ -141,10 +141,14 @@ def ptk_version_info():
 
 
 @functools.lru_cache(1)
-def ptk_version_is_supported():
+def ptk_above_min_supported():
     minimum_required_ptk_version = (1, 0)
     return ptk_version_info()[:2] >= minimum_required_ptk_version
 
+@functools.lru_cache(1)
```

Comment:
need another new line above this line to clear the flake failures

Reply:
Sorry about that. I'll change it and keep this in mind in the future. Thanks for point it out.